index.html

```diff
@@ -24,71 +24,449 @@
-                    <h1 class="head text">I’m Abubakar Glad to see you!</h1>
-                </section>
-                <section>
-                    <h3 class="description">
-                        I’m a software developer! I can help you build a product , 
-                        feature or website Look through some of my work and experience! 
-                        If you like what you see and have a project you need coded,
-                        don’t hestiate to contact me.
-                    </h3>
-                </section>
-                <section>
-                <h4 class="follow" >LET'S CONNECT</h4>
-                   <dl class="list container">
-                    <dt class="list-item">
-                        <a href="https://twitter.com/haywayalive" target="_blank" rel="noopener noreferrer"><i class="fa fa-twitter"></i></a>
-                    </dt>
-                    <dd></dd>
-                    <dt class="list-item">
-                        <a href="https://www.linkedin.com/in/abubakar-ummar-4b6643245" target="_blank" rel="noopener noreferrer"><i class="fa fa-linkedin"></i></a>
-                    </dt>
-                    <dd></dd>
-                    <dt class="list-item">
-                        <a href="https://medium.com/@haywayaheadshotstrategy.fx" target="_blank" rel="noopener noreferrer"><i class="fa fa-medium"></i></a>
-                    </dt>
-                    <dd></dd>
-                    <dt class="list-item">
-                        <a href="https://github.com/Haywayaheadshot" target="_blank" rel="noopener noreferrer"><i class="fa fa-github"></i></a>
-                    </dt>
-                    <dd></dd>
-                    <dt class="list-item">
-                        <a href="https://angel.co/u/abubakar-ummar" target="_blank" rel="noopener noreferrer"><i class="fa fa-angellist"></i></a>
-                    </dt>
-                    <dd></dd>
-                    
-                </dl>
-                
-                </section>
-          </main>
+  <body class="grid-container">
+    <div class="grid-item">
+      <div class="landing-page-overall-container">
+        <header class="landing-page-header">
+          <div class="logo-div">
+            <a class="logo">My Logo</a>
+          </div>
+          <nav class="hamburger-icon">
+            <a href="javascript:void(0);" class="icon" onclick="myFunction()">
+              <i class="fa fa-bars"></i>
+            </a>
+          </nav>
+        </header>
+
+        <h1 class="landing-page-main-head">I’m Abubakar Glad to see you!</h1>
+        <h3 class="landing-page-main-body-text">
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </h3>
+
+        <h4 class="follow">LET'S CONNECT</h4>
```

Comment:
- [ ] Kindly be noticed that The button `LET'S CONNECT` is `a` link a or `button` tag.